app/Console/Commands/ForceDeletePosts.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Illuminate\Console\Command;
+use Rogue\Managers\PostManager;
+
+class ForceDeletePosts extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:forceDeletePosts
+                            {type}
+                            {source}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Force deletes posts in Rogue by type and source';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(PostManager $postManager)
+    {
+        // Post Manager Instance.
+        $this->postManager = $postManager;
```

Comment:
Nitpicky, but if we're going to document this let's add the property & docblock (rather than creating implicitly)!

```php
/**
 * The post manager.
 * @var PostManager
 */
protected $postManager;
```